src/app/organizations/settings/organization-billing.component.ts

```diff
@@ -30,4 +45,110 @@ export class OrganizationBillingComponent extends UserBillingComponent implement
       this.firstLoaded = true;
     });
   }
+
+  async load() {
+    if (this.loading) {
+      return;
+    }
+    this.loading = true;
+    if (this.organizationId != null) {
+      this.billing = await this.apiService.getOrganizationBilling(this.organizationId);
+    }
```

Comment:
Can we enter this component without an orgnaizationId?

Reply:
After reviewing the routing module/permissions for this component, it appears the answer to your question is no. This is just leftover for temporarily porting over the super class until the admin vault refresh.

Reply:
@vincentsalucci can you ensure we have a ticket for to ensure this gets completed in the admin refresh?